### ISSUE TITLE
Assert on perilous precipice during jump sequence.

### ISSUE THREAD

**Issue by Phanterm** [frogatto/frogatto#259]
![image](https://cloud.githubusercontent.com/assets/3878045/12773107/e8b5108e-c9fd-11e5-916c-562e1616b0ed.png)

**Comment by rkettering** [comment]
The basic issue here is that berd gets attached to Frogatto (parented) to let frogatto carry him (and to automate him following frogatto's sprite around).   However, **he still has a solid area**, and we can't realistically get rid of this because he does more than one animation whilst clinging there.

Generally speaking, we're looking for an anura enhancement here, and we can use this to eliminate one legacy feature of anura with related uses.  We currently have no easy way to create one specific tweak to an object:  to take an object which has a solid area, and to cause it to not only be in a different solid_dimension than any other objects (something that's already possible), but to also cause it to be in a different solid dimension than any terrain.

The main purpose of this is heavy convenience - we've got a bunch of objects where we've got limited/rare cases where we need to apply this, but we want to apply this without being restricted from using our existing animations (otherwise we'd have to specify a full set of new animations for the object which had no solid area).   Right now, we use a specific feature to do this for a few things like anthills making the ants that are in the process of spawning "intangible" - we use the `attached_objects` system.

The attached objects system kinda sucks in a bunch of ways - I'm not sure it's possible to custom-position child objects in it, and I'm not sure the child objects can do any animation or processing either.  If we can get rid of it, that would be really nice.
